core/capabilities/ccip/configs/evm/contract_reader.go

```diff
@@ -236,6 +243,51 @@ var SourceReaderConfig = evmrelaytypes.ChainReaderConfig{
 				},
 			},
 		},
+		consts.ContractNameFeeQuoter: {
+			ContractABI: fee_quoter.FeeQuoterABI,
+			Configs: map[string]*evmrelaytypes.ChainReaderDefinition{
+				consts.MethodNameFeeQuoterGetStaticConfig: {
+					ChainSpecificName: mustGetMethodName("getStaticConfig", feeQuoterABI),
+					ReadType:          evmrelaytypes.Method,
+				},
+				consts.MethodNameFeeQuoterGetTokenPrices: {
+					ChainSpecificName: mustGetMethodName("getTokenPrices", feeQuoterABI),
+					ReadType:          evmrelaytypes.Method,
+				},
+				consts.MethodNameFeeQuoterGetTokenPrice: {
+					ChainSpecificName: mustGetMethodName("getTokenPrice", feeQuoterABI),
+					ReadType:          evmrelaytypes.Method,
+				},
+				consts.MethodNameGetFeePriceUpdate: {
+					ChainSpecificName: mustGetMethodName("getDestinationChainGasPrice", feeQuoterABI),
+					ReadType:          evmrelaytypes.Method,
+				},
+				consts.MethodNameGetDestChainConfig: {
+					ChainSpecificName: mustGetMethodName("getDestChainConfig", feeQuoterABI),
+					ReadType:          evmrelaytypes.Method,
+				},
+				consts.MethodNameGetPremiumMultiplierWeiPerEth: {
+					ChainSpecificName: mustGetMethodName("getPremiumMultiplierWeiPerEth", feeQuoterABI),
+					ReadType:          evmrelaytypes.Method,
+				},
+				consts.MethodNameGetTokenTransferFeeConfig: {
+					ChainSpecificName: mustGetMethodName("getTokenTransferFeeConfig", feeQuoterABI),
+					ReadType:          evmrelaytypes.Method,
+				},
+				consts.MethodNameProcessMessageArgs: {
+					ChainSpecificName: mustGetMethodName("processMessageArgs", feeQuoterABI),
+					ReadType:          evmrelaytypes.Method,
+				},
+				consts.MethodNameGetValidatedTokenPrice: {
+					ChainSpecificName: mustGetMethodName("getValidatedTokenPrice", feeQuoterABI),
+					ReadType:          evmrelaytypes.Method,
+				},
+				consts.MethodNameGetFeeTokens: {
+					ChainSpecificName: mustGetMethodName("getFeeTokens", feeQuoterABI),
+					ReadType:          evmrelaytypes.Method,
+				},
+			},
+		},
```

Comment:
Could you limit this to the minimum required methods? That will make things a little easier for non-EVM integrations.

Reply:
good point. let me do it